DUI3-DX/DUI3/Speckle.Connectors.DUI/Models/Card/SenderModelCard.cs

```diff
@@ -1,12 +1,11 @@
 using Speckle.Connectors.DUI.Models.Card.SendFilter;
-using Speckle.Newtonsoft.Json;
 
 namespace Speckle.Connectors.DUI.Models.Card;
 
 public class SenderModelCard : ModelCard
 {
   public ISendFilter? SendFilter { get; set; }
 
-  [JsonIgnore]
-  public HashSet<string> ChangedObjectIds { get; set; } = new();
+  // [JsonIgnore]
```

Comment:
happy to get rid of from it. You can remove it I believe. God bless `EvictObjects`